partiql-lang/src/main/kotlin/org/partiql/lang/eval/Bindings.kt

```diff
@@ -56,6 +56,63 @@ fun PartiqlAst.CaseSensitivity.toBindingCase(): BindingCase = when (this) {
     is PartiqlAst.CaseSensitivity.CaseSensitive -> BindingCase.SENSITIVE
 }
 
+/**
+ * Represents a namespaced global binding.
+ */
+class BindingId(
+    private val qualifier: List<BindingName>,
+    private val id: BindingName
```

Comment:
As I mentioned in a comment in the PIG model--the corollary here is that there should just be one list of `BindingName`, but if for some reason you don't end up going that route:

- `id` -> `head` or `root` (pick one, remain consistent)
- `qualifier` should be the 2nd argument, not the first